baker/sitemap.ts

```diff
@@ -74,6 +75,7 @@ export const makeSitemap = async (
         (postrow) => !alreadyPublishedViaGdocsSlugsSet.has(postrow.slug)
     )
     const gdocPosts = await db.getPublishedGdocPosts(knex)
+    const authorPages = await GdocAuthor.getPublishedAuthors(knex)
```

Comment:
You should just use `getMinimalAuthors` here because we only need the slug and title. `getPublishedAuthors` is quite heavy as it has to load images and documents etc.